tornado_rest_client/api.py

```diff
@@ -125,7 +125,7 @@ def wrapper(self, *args, **kwargs):
                     # pop it before searching.
                     default_exc = exc_conf.pop('', False)
                     log.debug('Searching through %s' % exc_conf)
-                    matched_exc = [exc for key, exc in exc_conf.items()
+                    matched_exc = [exc for key, exc in list(exc_conf.items())
```

Comment:
FYI: `dict.items()` is an iterator in python3, but a list in python2.  the automatic conversion tools are conservative and slap an unnecessary list coercion in places like this.